cpp/ql/src/Security/CWE/CWE-120/OverrunWrite.c

```diff
@@ -1,9 +1,9 @@
-void sayHello()
+void sayHello(uint32_t userId)
 {
-	char buffer[10];
+	char buffer[18];
 
-	// BAD: this message overflows the buffer
-	strcpy(buffer, "Hello, world!");
+	// BAD: this message overflows the buffer if userId >= 10000
+	sprintf(buffer, "Hello, user %d!", userId);
```

Comment:
I see, the original example would be a result for `VeryLikelyOverrunWrite.c` not `OverrunWrite.c` now. :+1: